.github/workflows/issue-metrics.yml

```diff
@@ -0,0 +1,42 @@
+name: Monthly Issue Metrics
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "3 2 1 * *"
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: issue metrics
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: read
+    steps:
+      - name: Get dates for last month
+        shell: bash
+        run: |
+          # Calculate the first day of the previous month
+          first_day=$(date -d "last month" +%Y-%m-01)
+
+          # Calculate the last day of the previous month
+          last_day=$(date -d "$first_day +1 month -1 day" +%Y-%m-%d)
+
+          #Set an environment variable with the date range
+          echo "$first_day..$last_day"
+          echo "last_month=$first_day..$last_day" >> "$GITHUB_ENV"
+
+      - name: Run issue-metrics tool
+        uses: github/issue-metrics@v3
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SEARCH_QUERY: 'repo:bpftrace/bpftrace created:${{ env.last_month }} -reason:"not planned"'
+
+      - name: Create issue
```

Comment:
I wouldn't create issue spam, that will make everyone hate this. Maybe make this the output of the job: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#adding-a-job-summary

I think this would be a custom step that does `cp issue_metrics.md $GITHUB_STEP_SUMMARY`

Reply:
You would just go to "Actions" and click this specific workflow.

I don't have hard objections to the issue, but maybe let's just start by generating and clicking, and if it's a good/useful report we can make it publish.

Reply:
Ok fine. I can do that.